plugins/woocommerce/changelog/add-41586

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: add
+
+Introduce a product type selection within the new experience
```

Comment:
Should mention that this is the new **product editor** experience

Reply:
done -> https://github.com/woocommerce/woocommerce/pull/41823/commits/ef7108cb02d1e54eb8e0c6166a9b81c55745d6ff